src/components/SelectionList/types.ts

```diff
@@ -233,6 +233,9 @@ type BaseSelectionListProps<TItem extends User | RadioItem> = Partial<ChildrenPr
 
     /** Whether to show the loading indicator for new options */
     isLoadingNewOptions?: boolean;
+
+    /** Fired when the list is displayed with the items */
+    onLayout?: (event: LayoutChangeEvent) => void;
```

Comment:
This prop appears unused, is it safe to remove?

Reply:
https://github.com/Expensify/App/blob/9463e22ba5c0897d519c0ccc85c12ae591742541/src/pages/SearchPage/index.js#L161-L162